### PR TITLE
add Semeru/OL tags

### PR DESCRIPTION
**Pull request by dmuelle**
#1566

### CODE REVIEW

File: blog_tags.json [hunk 1]
```diff
@@ -384,7 +385,7 @@
         },
         {
             "name": "performance enhancements",
-            "posts": ["mpJWT-performance-improvements",
+            "posts": ["open-liberty-semeru-performance","mpJWT-performance-improvements",
```
Comment (mbroz2) [inline blog_tags.json]:
```suggestion
            "posts": ["open-liberty-semeru-performance", "mpJWT-performance-improvements",
```

File: blog_tags.json [hunk 2]
```diff
@@ -323,7 +324,7 @@
         },
         {
             "name": "developer experience",
-            "posts": ["liberty-tools-eclipse",
+            "posts": ["open-liberty-semeru-performance","liberty-tools-eclipse",
```
Comment (mbroz2) [inline blog_tags.json]:
```suggestion
            "posts": ["open-liberty-semeru-performance", "liberty-tools-eclipse",
```

**Review by mbroz2 — DISMISSED** [review]
I prefer spaces separating each tag, but it doesn't matter so approved either way.

**Comment by dmuelle** [comment]
added spaces and will do the same going forward

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**